code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1584,7 +1584,7 @@
 		return 1
 
 	var/datum/butchering_product/teeth/T = locate(/datum/butchering_product/teeth) in src.butchering_drops
-	if(T && T.amount >= 2)
+	if(T && T.amount >= -1)
 		return 1
 
 	return 0
```

Comment:
you can probably just replace lines 1586-1590 with just ```return 1``` if the amount of teeth doesnt matter anymore